pull_request_template.md

```diff
@@ -1,4 +1,9 @@
+Thank you for taking the time to contribute to this library.
+
 ### Check List Before Merging! (For the Author)
 - [ ] Story Book has been rebuilt (new build artifacts in `/docs`)
 - [ ] Components have been rebuilt (new build artifacts in `/dist`)
 - [ ] All new components have been exported in `/index.js`
+- [ ] Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/draft_release_notes.md) to document your changes)
+- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated with new component and all instances of that component in USASpending. Refer to inventory documentation [here](https://airtable.com/tblHeQOwctk5z5FOj/viw2T0ewpuDPOhHtL?blocks=show) for help with identifying instances of components. `if applicable`
+
```

Comment:
Should we add checkbox for broker similar to the Component Library Integration Status Report?

Reply:
I think the integration report should account for both broker and USAS.